tests/Fixtures/Integration/misc/PHP7_3.test

```diff
@@ -103,3 +112,10 @@ rand($a, $b, ); // `random_api_migration` rule
 settype($foo, "integer", ); // `set_type_to_cast` rule
 in_array($b, $c, ); // `strict_param` rule
 foo($a === null, ); // `yoda_style` rule
+$a =(unset)$z; // `no_unset_cast` rule
```

Comment:
didn't you pointed out it's pointless to add this rule here ?

Reply:
yeah, but could do no harm to prove that here by test?